concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1229,7 +1229,7 @@ Hale-1973-1798-1227	Human_Mind_and_Conditions-115	give back	v	10b1.42	85	GIVE BA
 Hale-1973-1798-1228	Human_Mind_and_Conditions-116	give in	v	10b1.43		
 Hale-1973-1798-1229	Human_Mind_and_Conditions-117	give up	v	10b1.44	1696	SURRENDER
 Hale-1973-1798-1230	Human_Mind_and_Conditions-118	greet	v	10b1.45	3552	GREET
-Hale-1973-1798-1231	Human_Mind_and_Conditions-119	guard	v	10b1.46		
+Hale-1973-1798-1231	Human_Mind_and_Conditions-119	guard	v	10b1.46	1940	GUARD
```

Comment:
`1940 GUARD` is a noun in concepticon, so would unmap. https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L1941